### PR TITLE
Add a Transformer to format a Unix timestamp as a human-readable string.

### LINKED ISSUE #58: Create Transformer to format a Unix timestamp as a textual date string
I have this code. Working on tests and documentation.

### PR DESCRIPTION
**Pull request by kdebisschop**
Resolves #58

### CODE REVIEW

File: src/Transformers/FormatUnixTime.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
```
Comment (ecourtial) [inline src/Transformers/FormatUnixTime.php:4]: Little glitch here :)

File: tests/Transformers/FormatUnixTimeTest.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * @author      Wizacha DevTeam <dev@wizacha.com>
+ * @copyright   Copyright (c) Wizacha
+ * @copyright   Copyright (c) Leonardo Marquine
```
Comment (ecourtial) [inline tests/Transformers/FormatUnixTimeTest.php:6]: The contribution comes from the original project ?

**Review by ecourtial — CHANGES_REQUESTED** [review]
Two small issues in the doc

File: src/Transformers/FormatUnixTime.php (hunk 1 above)
Comment (kdebisschop) [reply src/Transformers/FormatUnixTime.php]: Fix pushed. Thanks.

File: tests/Transformers/FormatUnixTimeTest.php (hunk 2 above)
Comment (kdebisschop) [reply tests/Transformers/FormatUnixTimeTest.php]: Fix pushed. Thanks.

**Review by ecourtial — APPROVED** [review]

**Review by Nicolas-Masson-Wizaplace — APPROVED** [review]

**Pull request merged**